policies/lib/pods_test.rego

```diff
@@ -0,0 +1,91 @@
+package lib.pods
```

Comment:
do we want any test files from a vendor here? They already test what we implement, so I'd say remove these.

Reply:
ok, the `lib` folder is basically the entire konstraint library. I can remove the test files, they serve no purpose.